packages/cerebral-provider-firebase/src/remove.js

```diff
@@ -1,13 +1,14 @@
 import {
   createRef
 } from './helpers'
+import {FirebaseProviderError} from './errors'
 
 export default function remove (path) {
   const ref = createRef(path)
-  return new Promise((resolve, reject) => {
-    ref.remove().then(
-      () => resolve({}),
-      (error) => reject({error: error.message})
-    )
-  })
+
+  return ref.remove()
+    .then(() => undefined)
```

Comment:
Can use `export const noop = () => {}` from utils for this purpose?
Good point on being explicit, but returning undefined is dangerous in fact.